### PR TITLE
perception: moving lazy_free_space_updater into it's own library

### PR DESCRIPTION
**Pull request by jbohren**
This is so it can be used in another filter without confounding library deps.

### CODE REVIEW

**Comment by isucan** [comment]
+1

File: perception/depth_image_octomap_updater/CMakeLists.txt [hunk 1]
```diff
@@ -1,13 +1,10 @@
 set(MOVEIT_LIB_NAME moveit_depth_image_octomap_updater)
 
 add_library(${MOVEIT_LIB_NAME}_core
-  src/lazy_free_space_updater.cpp
   src/depth_image_octomap_updater.cpp
   )
 
-set_target_properties(${MOVEIT_LIB_NAME}_core PROPERTIES COMPILE_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
-set_target_properties(${MOVEIT_LIB_NAME}_core PROPERTIES LINK_FLAGS "${OpenMP_CXX_FLAGS}")
```
Comment (mikeferguson) [inline perception/depth_image_octomap_updater/CMakeLists.txt:9]: Are you sure that these FLAGS should be removed and only applied to the new library?

**Comment by davetcoleman** [comment]
+1

File: perception/depth_image_octomap_updater/CMakeLists.txt (hunk 1 above)
Comment (jbohren) [inline perception/depth_image_octomap_updater/CMakeLists.txt:9]: Yes. There's no openmp stuff in the `depth_image_octomap_updater`

**Comment by jbohren** [comment]
Shall we merge this?

File: perception/depth_image_octomap_updater/CMakeLists.txt (hunk 1 above)
Comment (mikeferguson) [inline perception/depth_image_octomap_updater/CMakeLists.txt:9]: Thanks for confirmation!

**Pull request merged**